sdk/attestation/azure-security-attestation/src/attestation_administration_client.cpp

```diff
@@ -75,6 +75,9 @@ AttestationAdministrationClient::GetAttestationPolicy(
     GetPolicyOptions const& options,
     Azure::Core::Context const& context) const
 {
+  AZURE_ASSERT_MSG(
```

Comment:
I wonder if we should add `AZURE_ASSERT_FALSE_MSG` macro instead of using `!` for the condition.
For example, we have `AZURE_ASSERT_FALSE(exp)`.... 
It might make it easier to read the expressions

Reply:
Honestly, I'm not sure it helps readability. !foo.empty() shows up a lot in STL based programming for checks to see if something has values or not. So it should be a fairly familiar pattern.